examples/logging-prometheus/custom-fn.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+from typing import List
+
+def mean_pixel_red(img: np.ndarray):
+    assert(img.shape[2] == 3)
```

Comment:
Being super nitpicky here, but a good rule is in general to raise errors instead of assertions : 
https://snyk.io/blog/the-dangers-of-assert-in-python/#:~:text=TL%3BDR%20%E2%80%94%20Assert%20statements%20are,which%20can%20cause%20production%20outages.
Also, would be nice to have some error message so the user knows what they are doing wrong